scheduler/src/cook/config.clj

```diff
@@ -395,7 +395,11 @@
                                          {:base-calculator (config-string->fitness-calculator
                                                             (get data-local-fitness-calculator :base-calculator  "com.netflix.fenzo.plugins.BinPackingFitnessCalculators/cpuMemBinPacker"))
                                           :data-locality-weight (get data-local-fitness-calculator :data-locality-weight 0.95)
-                                          :maximum-cost (get data-local-fitness-calculator :maximum-cost 100)})}))
+                                          :cost-endpoint (get data-local-fitness-calculator :cost-endpoint nil)
+                                          :batch-size (get data-local-fitness-calculator :batch-size 500)
+                                          :update-interval-ms (get data-local-fitness-calculator :update-interval-ms nil)
+                                          :launch-wait-seconds (get data-local-fitness-calculator :launch-wait-seconds 60)
+                                          :cache-ttl-ms (get data-local-fitness-calculator :cache-ttl-ms 300000)})}))
```

Comment:
Nitpick: lexicographic order

Reply:
We should consider using destructuring on `data-local-fitness-calculator` with defaults provided in the `or` clause.